src/components/backdrop/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why did these folders get `package.json` files?

Reply:
That's one of the ways webpack is able to resolve a specific module. When your have multiple js files inside a dir and only one is required by site.js. then the main js file requires the rest of the files in that dir. Ir could be other types of files inside a dir. The other way of resolving is defining a module in webpack.config with the exact path to it.

Reply:
When you have a direct with multiple files inside that are required by site.js, I map that directory as a possible modules path that webpack can look for modules.